### ISSUE TITLE
[QuantNoise] Details of int4 quantization in image classification task

### ISSUE THREAD

**Issue by hey-yahei** [pytorch/fairseq#2383]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?       
Thanks for open-sourcing such a great project!    
    
I'm tring to reproduce QuantNoise for int4-mobilenetv2 in ImageNet according to the paper and this project. But I'm confused about some details in the paper. 
1. ***Table1*** in the paper shows that training int4-EfficientNet-B3 in ImageNet-1k with QAT and Quant-Noise gets 59.4% and 67.8% top-1 accuracy respectively. In such a experiment(Table1), 
    1.  ***Section8.3*** say that activations are quantized with Hitogram(L2) scheme in scalar quantization.      
        **What about weights? Are weights all quantized with Histogram(L2) scheme as well?** 
    2. As I know, origin QAT quantized activations with MovingAverageMinMax scheme and fold BN into Conv to apply FakeBN.      
        **Which scheme is used for QAT in the experiment, Histogram(L2) or MovingAverageMinMax?**      
        **Does QAT use FakeBN in the experiment, or just use normal BN? Does the normal BN freeze statistic parameters(running_mean and running_varience)?**
    3. **Are all weights and activations symmetricly quantized into int4, including depthwise convolutions, downsample convolutions, SE modules, the first convolution as well as the last fully connection layer?**
2. ***Table9*** compares the results of Histogram and Channel scheme.     
    **Are weights and activations both quantized with Histogram scheme or Channel scheme here?**

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by MohammedHAlali** [comment]
Hi @hey-yahei, did you figure out how to run the image classification task on MobileNet?
